Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -5,6 +5,37 @@ protocol PhoneNumberTextFieldDelegate: AnyObject {
     func phoneNumberIsUpdated(number: String, tag: Int)
 }
 
+class BorderView: UIView {
+
```

Comment:
We need to unit test `BorderView`.  Some ideas:
```swift
func testInitDoesNotAddLayer()
func testLayoutSubviewsDoesNotAddLayerIfNoStyle()

// These could be merged into 1 if we force the layer creation when style is provided
func testUpdateStyleDoesNotAddLayer()
func testLayoutSubviewsAddsLayerIfStylePresent()

// Suggested to update color internally and not expose directly the layer reference
func testBorderColorUpdate()
```

The layer can always be found via `view.layer.sublayers`, which we can use to test these behaviours.